internal/file/file_plugin.go

```diff
@@ -301,8 +297,13 @@ func (fp *FilePlugin) createDataPlaneResponse(correlationID string, status mpi.C
 		CommandResponse: &mpi.CommandResponse{
 			Status:  status,
 			Message: message,
-			Error:   err,
 		},
 		InstanceId: instanceID,
 	}
+
+	if err != "" {
+		response.GetCommandResponse().Error = err
```

Comment:
this seems unusual. Why is this being unset here?

Reply:
If there is an error it is being set in the response, can switch back to always setting it but seemed a bit odd to set the err if there was none

Reply:
Reverted the change